app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+
+  def format_errors(object)
+    "#{object.errors.messages.values.join(' ')}"
```

Comment:
I haven't used `full_messages` cause Oauth_uid was appearing in the sentence flashed to the user